packages/expo-network/src/ExpoNetwork.web.ts

```diff
@@ -0,0 +1,27 @@
+import { NetworkState, NetworkStateType } from './Network.types';
+
+export default {
+  async getIpAddressAsync(): Promise<string> {
+    return new Promise((resolve, reject) => {
```

Comment:
you should be able to use `await` in this method instead of the various `.then` calls